cmd/authelia-scripts/cmd_suites.go

```diff
@@ -143,6 +143,10 @@ func runSuiteSetupTeardown(command string, suite string) error {
 	cmd.Stderr = os.Stderr
 	cmd.Env = os.Environ()
 
+	if selectedSuite == suitePathPrefix {
```

Comment:
OPT: I think this could be improved by generalizing the concept of loading env var in any suite. It could be something like reading env variables from a file stored in the suite resources for instance.

Reply:
Addressed in https://github.com/authelia/authelia/pull/1052/commits/84eb500267d29895b0a6b0daaff3af36f31d22c1.